manage_people/migrations/0001_mp_initial.py

```diff
@@ -20,71 +21,88 @@
-	('manage_people', 'hls', 'TeacherEnrollment', False),
-	('manage_people', 'hls', 'Teaching Staff', False)
+    ('manage_people', '*', 'Course Head', True),
+    ('manage_people', '*', 'Course Support Staff', True),
+    ('manage_people', '*', 'Department Admin', True),
+    ('manage_people', '*', 'DesignerEnrollment', True),
+    ('manage_people', '*', 'Faculty', True),
+    ('manage_people', '*', 'Guest', False),
+    ('manage_people', '*', 'Help Desk', True),
+    ('manage_people', '*', 'Librarian', True),
+    ('manage_people', '*', 'ObserverEnrollment', False),
+    ('manage_people', '*', 'SchoolLiaison', True),
+    ('manage_people', '*', 'StudentEnrollment', False),
+    ('manage_people', '*', 'TaEnrollment', True),
+    ('manage_people', '*', 'TeacherEnrollment', True),
+    ('manage_people', '*', 'Teaching Staff', True),
+    ('manage_people', 'gse', 'Help Desk', False),
+    ('manage_people', 'hks', 'Course Head', False),
+    ('manage_people', 'hks', 'Course Support Staff', False),
+    ('manage_people', 'hks', 'DesignerEnrollment', False),
+    ('manage_people', 'hks', 'Librarian', False),
+    ('manage_people', 'hks', 'TaEnrollment', False),
+    ('manage_people', 'hks', 'Teaching Staff', False),
+    ('manage_people', 'hls', 'Account Observer', False),
+    ('manage_people', 'hls', 'Course Head',  False),
+    ('manage_people', 'hls', 'Course Support Staff', False),
+    ('manage_people', 'hls', 'Department Admin', False),
+    ('manage_people', 'hls', 'DesignerEnrollment', False),
+    ('manage_people', 'hls', 'Faculty', False),
+    ('manage_people', 'hls', 'Help Desk', False),
+    ('manage_people', 'hls', 'Librarian', False),
+    ('manage_people', 'hls', 'TaEnrollment', False),
+    ('manage_people', 'hls', 'TeacherEnrollment', False),
+    ('manage_people', 'hls', 'Teaching Staff', False)
 ]
 
 
-
 def populate_manage_people_role(apps, schema_editor):
     ManagePeopleRole = apps.get_model('manage_people', 'ManagePeopleRole')
     fields = ('user_role_id', 'canvas_role_label', 'xid_allowed')
     with transaction.atomic():  # wrap all the inserts in a transaction
         for values in MANAGE_PEOPLE_ROLE_DATA:
             ManagePeopleRole.objects.create(**dict(zip(fields, values)))
 
+
 def create_lti_permissions(apps, schema_editor):
-    LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    try:
+        LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    except LookupError:
+        # If the LtiPermission table doesn't exist, return from here
+        return
+
     fields = ('permission', 'school_id', 'canvas_role', 'allow')
 
     for permission in LTI_PERMISSIONS_DATA:
         LtiPermission.objects.create(**dict(zip(fields, permission)))
 
+
 def reverse_permissions_load(apps, schema_editor):
-    LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    try:
+        LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    except LookupError:
```

Comment:
It wasn't previously protected; if we can get the model, then we can use its object manager to filter and perform deletes. If there are problems (e.g. the model is defined but the table doesn't exist), they would've existed in the old version of the code too. I guess I lean towards throwing an error in that case, because something is unexpectedly wrong, than swallowing it.